packages/cicero-core/src/template.js

```diff
@@ -172,14 +179,85 @@ class Template {
         return hasher.digest('hex');
     }
 
+    /**
+     * verifies the signature stored in the template object using the template hash and timestamp
+     */
+    verifyTemplateSignature() {
+        const templateHash = this.getHash();
+        if (this.authorSignature === null) {throw new Error('The template is missing author signature!');}
+        const signature = this.authorSignature.templateSignature.signature;
+        const timestamp = this.authorSignature.templateSignature.timestamp;
+        const signatoryCert = this.authorSignature.templateSignature.signatoryCert;
+        //X509 cert converted from PEM to forge type
+        const certificateForge = forge.pki.certificateFromPem(signatoryCert);
+        //public key in forge typenode index.js sign acme 123 helloworldstate
```

Comment:
What does this comment mean? Is this a debugging artifact?

Reply:
Sorry I missed to remove that. Yes I was using that while debugging.

Reply:
Corrected the comment.